src/api/controllers/UserController.ts

```diff
@@ -1,22 +1,17 @@
 import {
-  JsonController, Params, Get, Post, Delete, UseBefore, Body, BodyParam,
+  Body, Delete, Get, JsonController, Params, Post,
 } from 'routing-controllers';
-import { UserModel } from '../../models/UserModel'
+import { UuidParam } from '../validators/GenericRequests';
 import { UserService } from '../../services/UserService';
-import { Inject } from 'typedi';
 import {
-  GetUsersResponse,
-  GetUserResponse,
   CreateUserRequest,
-  Uuid,
-  GenericSuccessResponse,
+  ErrorResponse,
+  GetUserByEmailRequest,
+  GetUserResponse,
+  GetUsersResponse,
+  getErrorMessage,
```

Comment:
how come this is theonly thing not capitalized first

Reply:
`getErrorMessage` is a function that returns the error in string format. Everything else is an Interface

Reply:
gotcha